sections/futures/EditPositionModal/EditStopLossAndTakeProfitModal.tsx

```diff
@@ -161,7 +192,7 @@ export default function EditStopLossAndTakeProfitModal() {
 			<StyledSpacer marginTop={6} />
 			<EditStopLossAndTakeProfitInput
 				type={'take-profit'}
-				invalid={false}
+				invalid={tpInvalid}
```

Comment:
Shall we also apply `tpInvalid` and `slInvalid` checks to enable/disable the `Set TP/SL` button?

Reply:
Yes good idea